helix-term/src/ui/editor.rs

```diff
@@ -978,6 +978,8 @@ impl EditorView {
     }
 
     pub fn handle_idle_timeout(&mut self, cx: &mut commands::Context) -> EventResult {
+        commands::compute_inlay_hints_for_all_views(cx.editor, cx.jobs);
```

Comment:
I am not 100% sure why but the inlay hints don't seem to update for me while scrolling with the mouse.
It seems that we do not reset the idle timeout on mouse actions. I think that might actually be an existing bug that would be fixed by adding:
``` rust
cx.editor.reset_idle_timer();
```
at the top of `handle_mouse_event`